pkg/template/event_test.go

```diff
@@ -414,6 +416,8 @@ func Test_ApplyEventToParams_error(t *testing.T) {
 }
 
 func Test_NewResources(t *testing.T) {
+	// This seeds Uid() to return 'dpms2'
+	rand.Seed(0)
```

Comment:
Shouldn't this line be moved into the `t.Run` function, so that `rand` will be seeded at the beginning of every `t.Run` test case?

Reply:
The library is only imported once and its internal state is seeded as specified. Seeding it again this way is probably a better practice though since the second value generated would be unknown (if `$(uid)` was used multiple times in the test).